src/software/thunderscope/binary_context_managers/game_controller.py

```diff
@@ -21,51 +21,80 @@
 
 
 class Gamecontroller(object):
-    """ Gamecontroller Context Manager """
+    """Gamecontroller Context Manager"""
 
     CI_MODE_LAUNCH_DELAY_S = 0.3
     REFEREE_IP = "224.5.23.1"
     CI_MODE_OUTPUT_RECEIVE_BUFFER_SIZE = 9000
 
-    def __init__(self, supress_logs: bool = False) -> None:
-        """Run Gamecontroller
+    def __init__(
+        self,
+        supress_logs: bool = False,
+        use_unconventional_port: bool = False,
+        not_launch_gc: bool = True,
```

Comment:
this shouldn't be passed here, this should be used for conditional initialization in `thunderscope_main`